src/java/main/org/apache/zookeeper/server/quorum/Leader.java

```diff
@@ -106,10 +104,10 @@ public String toString() {
     private final HashSet<LearnerHandler> learners =
         new HashSet<LearnerHandler>();
 
-    private final ProposalStats proposalStats;
+    private final BufferStats bufferStats;
```

Comment:
Could you use "proposalStats" variable name instead of "bufferStats", thats more readable.